crates/router/src/connector/aci.rs

```diff
@@ -152,7 +153,7 @@ impl
                 .get_connector_transaction_id()
                 .change_context(errors::ConnectorError::MissingConnectorTransactionID)?,
             "?entityId=",
-            auth.entity_id
+            auth.entity_id.peek()
```

Comment:
@SanchithHegde , since we're moving to secret, should the value be made available in (error) logs?